tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "experimentalDecorators": true,
     "emitDecoratorMetadata": true,
     "skipLibCheck": true,
-    "allowJs": true,
+    "allowJs": false,
```

Comment:
Why this is useful? Faster compilation?

Reply:
Just in general a preference to being more tight in what we accept (and getting software to verify it for us).